src/slack/routes.js

```diff
@@ -5,61 +5,82 @@ const router = express.Router();
 
 const slackbot = require('./api.js');
 
-const expectedQuoteObject = {
-  id: 1,
-  quote: 'Be inspired, friend! You are doing great!',
+const resourceServer = require('../quotes/generator.js');
+
+const expectedInspirationQuote = {
+  _id: 1,
+  content: 'Be inspired, friend! You are doing great!',
   author: 'me',
```

Comment:
I don't think we have author or img anymore. only the text/url as content and ids